config/Waystones.cfg

```diff
@@ -63,22 +63,22 @@ general {
     # The maximum xp cost when Blocks per XP Level is enabled.
     # Min: 1
     # Max: 2147483647
-    I:"Maximum XP Cost"=3
```

Comment:
It's liner, afaik. For every 500 blocks of distance, it would cost a level (with an additional 5 for jumping a dimension). With the removal of free teleports, it should always be one level.

Reply:
:+1: